backend/config.py

```
@@ -0,0 +1,276 @@
+Centralizes all environment variable loading and configuration management
+for improved code organization and maintainability.
+"""
+
+import os
+import secrets
+import string
+from pathlib import Path
+from typing import Optional
+
+
+class TransDockConfig:
```

Pylint (3.3.7)

[refactor] Too many instance attributes (33/7)

(R0902)